MTA-6889 CLI refactored commands (#372)
migtools/mta-documentation: docs/topics/mta-cli/proc_analyzing-single-app-with-mta-cli.adoc

[role="_abstract"]
To assess the effort required to migrate a specific application, you can analyze this application individually by using the {ProductFullName} command-line interface (CLI). 

> You can run an analysis to generate a report containing the details of the application migration effort, including potential migration issues. You can use the report to prioritize tasks and estimate the resources needed for the migration.

Reviewer: ```suggestion
You can run an analysis to generate a report that contains application migration details, including potential issues. You can use the report to prioritize tasks and estimate the resources that are needed for the migration.
```

Changed the participle "containing" to the finite relative clause "that contains", and changed "resources needed" to "resources that are needed" to make the sentence clearer for readers.

Minimalism and Fluff Removal (CQA 2.1 / IBM Style): Removed the repetition of the word "migration" in the phrase "potential migration issues". Since the context of the sentence has already established that this is about a migration report, repeating the word is unnecessary.

Author: I have made the changes, thanks! Please shall I retain the word "effort" in the para here? Because, migration effort is a numerical estimate and has a special meaning.
